CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Guide: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+## 2.9.0-rc.1
+
+**MapboxCommon**: v24.11.0-rc.2
+**MapboxCoreSearch**: v2.9.0-rc.2
```

Comment:
I checked that Android search 2.9.0-rc.1 depends on:
- Search Native SDK `2.9.0-rc.1`
- Common SDK `24.11.0-rc.1`

Did you investigate what are the right dependencies?